src/testnet/helium/node.rs

```diff
@@ -116,6 +140,38 @@ impl Node {
             event_dispatcher,
         }
     }
+
+    pub fn spawn_http_server(&mut self) {
+        // we can call _open_ here rather than _connect_, since connect is first called in
+        //   make_genesis_block
+        let mut burndb = BurnDB::open(&self.config.get_burn_db_path(), true)
+            .expect("Error while instantiating burnchain db");
+
+        let burnchain = Burnchain::new(&self.config.get_burn_db_path(),
+                                       &self.config.burnchain.chain, &self.config.burnchain.mode)
+            .expect("Error while instantiating burnchain");
+
+        let view = {
+            let mut tx = burndb.tx_begin().unwrap();
+            BurnDB::get_burnchain_view(&mut tx, &burnchain).unwrap()
+        };
+
+        // create a new peerdb
+        let peerdb = PeerDB::connect(
+            &self.config.get_peer_db_path(), true, TESTNET_CHAIN_ID, burnchain.network_id, i64::max_value() as u64,
+            UrlString::try_from(format!("http://{}", self.config.node.rpc_bind)).unwrap(),
+            &vec![], None).unwrap();
+        // use node = 0 for the chain state
+
+        let http_server = HttpPeer::new(TESTNET_CHAIN_ID, burnchain, view, DEFAULT_CONNECTION_OPTIONS.clone());
+        let socket_addr = self.config.node.rpc_bind.parse()
+            .expect(&format!("Failed to parse socket: {}", &self.config.node.rpc_bind));
+        let _join_handle = http_server.spawn(&socket_addr, peerdb, self.config.get_burn_db_path(),
+                                             self.config.get_chainstate_path(), false, TESTNET_CHAIN_ID, 5000)
+            .unwrap();
+
+        info!("Bound P2P/HTTP server on: {}", &self.config.node.rpc_bind);
```

Comment:
Except, the p2p server isn't getting started here?

Reply:
Updated message in 5951759a789563a6ecb1ae3800fc0706944b02ec